SphereEversion/ToMathlib/Analysis/InnerProductSpace/Dual.lean

```diff
@@ -21,10 +21,7 @@ theorem orthogonal_span_toDual_symm (π : E →L[ℝ] ℝ) :
   suffices (∀ a : ℝ, ⟪a • (toDual ℝ E).symm π, x⟫ = 0) ↔ π x = 0 by
     simp only [orthogonal, mem_mk, Set.mem_setOf_eq, LinearMap.mem_ker, ← toDual_symm_apply]
     change (∀ (u : E), u ∈ span ℝ {(LinearIsometryEquiv.symm (toDual ℝ E)) π} → inner u x = 0) ↔ _
-    simpa only [mem_span_singleton, forall_exists_index, forall_apply_eq_imp_iff, toDual_symm_apply]
```

Comment:
For this un-squeezing: can you at least roughly eyeball that performance didn't change?

Reply:
With `simp only [...]

````console
1.74s user 0.98s system 130% cpu 2.086 total
```

With `simpa` 

```console
1.77s user 1.00s system 126% cpu 2.193 total
```

Reply:
What do you think? Good enough?

Reply:
Good enough!